modules/outconBidAdapter.js

```diff
@@ -0,0 +1,53 @@
+import {registerBidder} from '../src/adapters/bidderFactory';
+import {config} from '../src/config';
+
+const BIDDER_CODE = 'outcon';
+export const spec = {
+  code: BIDDER_CODE,
+  isBidRequestValid: function(bid) {
+    return !!((bid.params.pod || (bid.params.internalId && bid.params.publisher)) && bid.params.env);
+  },
+  buildRequests: function(validBidRequests) {
+    for (let i = 0; i < validBidRequests.length; i++) {
+      let par = '';
+      let url = '';
+      if (validBidRequests[i].params.pod != undefined) par = 'get?pod=' + validBidRequests[i].params.pod;
+      else par = 'get?internalId=' + validBidRequests[i].params.internalId + '&publisher=' + validBidRequests[i].params.publisher;
+      switch (validBidRequests[i].params.env) {
+        case 'test':
+          par = par + '&demo=true';
+          url = 'http://test.outcondigital.com:8048/ad/' + par;
+          break;
+        case 'api':
+          url = 'http://api.outcondigital.com:8048/ad/' + par;
+          break;
+        case 'stg':
+          url = 'http://stg.outcondigital.com:8048/ad/' + par;
+          break;
+      }
+      return {
+        method: 'GET',
+        url: url,
+        data: {}
+      };
+    }
+  },
+  interpretResponse: function(serverResponse, bidRequest) {
+    const bidResponses = [];
+    const bidResponse = {
+      requestId: serverResponse.body.id,
+      cpm: serverResponse.body.cpm,
+      width: serverResponse.body.creatives[0].width,
+      height: serverResponse.body.creatives[0].height,
```

Comment:
This coment relates to both `width` & `height` fields.  In the test bid, the corresponding values under the `creatives` object were not populated.

For banner bids, these fields need to be populated with valid values or else Prebid will reject the bid later in the auction workflow.  Can you make the necessary updates on the test bid to ensure these values are present in the response?